reconfiguration/cmf/concord.cmf

```diff
@@ -106,6 +106,21 @@ Msg AddRemoveWithWedgeStatusResponse 31 {
     string config_descriptor
 }
 
+Msg UnwedgeStatusRequest 32{
+    uint64 sender
+}
+
+Msg UnwedgeStatusResponse 33{
```

Comment:
I would also add here a "reason" field that describes why the replica is unable to wedge